src/content/docs/infrastructure/host-integrations/host-integrations-list/snowflake-integration.mdx

```diff
@@ -57,8 +57,32 @@ binary](https://github.com/newrelic/newrelic-snowflake-integration/releases). Pl
     title="Step 3. Update `config.yaml`"
     >
 To ensure you're using the most recent information for the integration, update your `config.yaml` file so that the file matches the snippet found in Snowflake's [Key Pair authentication](https://github.com/newrelic/newrelic-snowflake-integration#key-pair-authentication) section. 
+    </Collapser>
+    <Collapser
+    id="step4"
+    title="Step 4. When the agent is running as a systemd service"
+    >      
+If running the New Relic infrastructure agent as a systemd service, follow these [steps to set the environment variable](https://github.com/newrelic/newrelic-snowflake-integration#when-the-agent-is-running-as-a-systemd-service)      
+i.To determine if you are on a system using `systemd` as the init service, run 
+```
+[[ `\systemctl` =~ -\.mount ]] && echo yes || echo no 
+```  
+    </Collapser>
+    <Collapser
+    id="step5"
+    title="Step 5. Setting NEWRELIC_SNOWFLAKE_HOME"
+    >
+Set the `NEWRELIC_SNOWFLAKE_HOME` environment variable as documented in the [Setting NEWRELIC_SNOWFLAKE_HOME](https://github.com/newrelic/newrelic-snowflake-integration#required-environment-variables) section. (Skip this step if you are running the New Relic infrastructure agent as a systemd service and followed the alternative steps for this).
+    </Collapser> 
+    <Collapser
+    id="step6"
+    title="Step 6. Update `flex config`"
+    >
+Copy the relevant flex config for your platform from [flexConfigs](https://github.com/newrelic/newrelic-snowflake-integration/tree/main/flexConfigs) to the agent's integrations.d folder.
+* for Linux, it is found at `/etc/newrelic-infra/integrations.d/`
+* for Windows, it is found at `C:\Program Files\New Relic\newrelic-infra\integrations.d\`.
 
-Once you've updated the YAML file, return to this doc to install the <InlinePopover type="dashboards" /> to your New Relic account. 
+Once you've followed all this steps, return to this doc to install the <InlinePopover type="dashboards" /> to your New Relic account.
```

Comment:
Once you've followed all the above steps, return to this doc to install the <InlinePopover type="dashboards" /> to your New Relic account.